src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,25 +77,31 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
       this.stopPolling();
 
-      // Wait 1 min before polling again & do not show error message when encounter rate limit error
-      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
-        error.responseJSON?.errorCode === 'E0000047') {
+      if (error.responseJSON?.errorSummaryKeys?.includes('idx.session.expired')) {
+        // Do NOT resume polling since session is invalid and polling is already stopped
+        return;
+      }
```

Comment:
Is this the only error for which we need to stop polling? Could there possibly be other errors from backend after which we may have to stop polling as well?

Reply:
@anipendakur-okta if you look at the cod.. polling is stopped first.. and then we check we have to resume it or not further down the function

Reply:
Right, I got that, My question was could there possibly be more errors with OR conditions to `return` since this is newly added code -
```
if (error.responseJSON?.errorSummaryKeys?.includes('idx.session.expired') || <any other error conditions to check?> ) {
  // Do NOT resume polling since session is invalid and polling is already stopped
  return;
}
```

Reply:
basically you want to say cases where we do not want to resume polling... The only case i am aware of is when the session is expired. But that is good question for Devices team to explore..